src/data-access/repositories/interfaces/manifest-repository.ts

```diff
@@ -0,0 +1,5 @@
+import { ActionManifest } from '../../../model/entities/action'
+
+export interface ManifestRepository {
```

Comment:
You are fetching `ActionManifests`, so atleast call the interface `ActionManifestRepository`.